src/easylink/step.py

```diff
@@ -104,8 +104,19 @@ def __init__(
         during the process of flattening the ``Stepgraph``, e.g. unrolling loops, etc. 
         For example, if step 1 is looped multiple times, each node would have a 
         ``step_name`` of, perhaps, "step_1" but unique ``names`` ("step_1_loop_1", etc)."""
+
+        if len(set(slot.name for slot in input_slots)) != len(input_slots):
+            raise ValueError(f"{step_name} has duplicate input slot names!")
```

Comment:
Shouldn't we log out the duplicates?

Reply:
As discussed real time, we won't do too much work to make these messages nice, since they are really only for us (pipeline schema developers)